bastionzero/zli
# Disambiguate targets now ignores terminated and errored targets

## Description of the change

In this PR: 
 - Disambiguate targets now ignores terminated and errored targets
 - New Status flag for lt command (-s online offline etc)
 - Refactor of silent flag to quiet

Testing:
 - Autodiscover two new targets with the exact same name (Manual Name mode)
 - Edit one of the targets in DB to be in Error state
 - Try to connect using `zli connect user@target` and you should be able to connect to the one listed as Online
 - Old zli will give you "Multiple targets detected with same name error"

## Type of change


## Related JIRA tickets

Relates to JIRA: CWC-653

## Potential Security Impacts

None, this is a client side change of UX.

## Checklists

### Development
